src/compat/browser_compatibility_types.ts

```diff
@@ -126,7 +126,9 @@ interface ICompatAudioTrackList extends EventTarget {
  */
 interface ICompatAudioTrack {
   id : string;
-  kind : string;
+  kind : "" | "alternative" | "main"
+            | "main-desc" | "translation"
+            | "commentary" | "descriptions";
```

Comment:
Maybe we should add a `?` and a `| undefined`? I say that because it seems to only be present in Safari and IE10 (wait what?).

Reply:
(Link: https://developer.mozilla.org/en-US/docs/Web/API/AudioTrack/kind#browser_compatibility)